README.md

```diff
@@ -76,7 +82,7 @@ For [Aave V2](./src/adapters/migration/AaveV2MigrationAdapter.sol), [Aave V3](./
 - Make use of transient storage.
 - Bundler3 is now a call dispatcher that does not require any approval.
   Because call-dispatch and approvals are now separated, it is possible to add adapters over time without additional risk to users of existing adapters.
-- All generic features are now in `GeneralAdapter1`, instead of being in separate files that are then all inherited by a single contract.
+- All generic features except ERC20 wrap and unwrap are now in `GeneralAdapter1`, instead of being in separate files that are then all inherited by a single contract.
```

Comment:
```suggestion
- All generic features are now in `GeneralAdapter1`, instead of being in separate files that are then all inherited by a single contract.
```